.editorconfig

```diff
@@ -0,0 +1,22 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.json]
+[*.{json,yaml,yml}]
+indent_size = 2
+
+[Makefile]
+indent_style = tab
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.neon]
+indent_style = tab
```

Comment:
All files should end with a blank line (can be enabled in the IDE).

Reply:
added in [refactor] suggested changes